node/config/def.go

```diff
@@ -69,7 +69,7 @@ func DefaultBoost() *Boost {
 			// TODO: It'd be nice to set this based on sector size
 			MaxDealStartDelay:               Duration(time.Hour * 24 * 14),
 			ExpectedSealDuration:            Duration(time.Hour * 24),
-			PublishMsgPeriod:                Duration(time.Hour),
+			PublishMsgPeriod:                Duration(time.Second * 10),
```

Comment:
Should we override this in the test instead?

Reply:
We should, actually I am already doing that, but it doesn't seem to have an effect... 😕